contracts/interchain-token-service/src/contract/execute.rs

```diff
@@ -60,6 +63,10 @@ pub fn execute_message(
             let destination_address = load_its_contract(deps.storage, &destination_chain)
                 .change_context_lazy(|| Error::UnknownChain(destination_chain.clone()))?;
 
+            apply_balance_invariant(deps.storage, &message, cc_id.source_chain.clone(), true)?;
+
+            apply_balance_invariant(deps.storage, &message, destination_chain.clone(), false)?;
```

Comment:
this looks a bit odd. Maybe combine this into one function call, and just pass in the source chain and destination chain, to avoid the `true` and `false`? The boolean parameters are quite cryptic